docs/getting_started/access_account.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Accessing your Custodian account
+sidebar_position: 4
+---
+ 
+In the custodian account, we only display those for which we (the shard admin) are key custodians. With your custodian account, you can create a key pair, add public keys and top up your tokens.
```

Comment:
```suggestion
In custodial accounts, we only display those accounts for which the Calimero platform is the custodian. With your custodian account, you can create a key pair, add public keys and top up your tokens.
```
The custodian is the Calimero deployment, not the shard admin, nor Calimero as a company.
Custody is in the k8s cluster, in the shard namespace.